### ISSUE TITLE
support $REDIS_HOST_PASSWORD_FILE

### ISSUE THREAD

**Issue by catniptwinz** [nextcloud/docker#1272]
Related to https://github.com/nextcloud/docker/issues/1179 and https://github.com/nextcloud/docker/pull/1232, when using Docker swarm secrets to manage Redis credentials, it's helpful to be able to use the same syntax in docker-compose that we already use for NextCloud and db credentials. For example:
```
environment:
  REDIS_HOST_PASSWORD_FILE: /run/secrets/redis_password
```
Enabling this appears to be as simple as calling `file_env REDIS_HOST_PASSWORD` at the appropriate point in entrypoint.sh. I've tested this with my own (php-fpm) installation and it appears to be working as expected. Would the maintainers be open to a PR that implements this change?

**Comment by schklom** [comment]
Even without Docker swarm you can use secrets in docker-compose (i think requires v3).
So it would be better for everyone exposing Nextcloud to the internet if the redis password could be implemented as a secret.

In `redis` container, it is possible to use a secret for the password but currently requires a little trick detailed [here](https://github.com/docker-library/redis/issues/46#issuecomment-335326234) or [here](https://github.com/docker-library/redis/issues/46#issuecomment-364761650).
Would the maintainers at least document how to do something similar ? `nextcloud`'s code wouldn't even need to be changed in that case, just the documentation.

**Comment by catniptwinz** [comment]
Looks as if #1325 is equivalent to what I'd have submitted and should address this issue.

**Comment by J0WI** [comment]
Fixed in #1325
